### PR TITLE
Update apiview-review-gen-swagger.yml

### PR DESCRIPTION
**Pull request by maririos**
FYI @chidozieononiwu

### CODE REVIEW

File: eng/pipelines/apiview-review-gen-swagger.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
   - script: >
       dotnet tool install
       Azure.Sdk.Tools.SwaggerApiParser
-      --version 1.0.7-dev.20230211.1
+      --version 1.0.7-dev.20230209.5
```
Comment (chidozieononiwu) [inline eng/pipelines/apiview-review-gen-swagger.yml:30]: This seems more like a downgrade than an upgrade.

Comment (maririos) [reply eng/pipelines/apiview-review-gen-swagger.yml:30]: I copied the one from the pipeline. Looking

Comment (maririos) [reply eng/pipelines/apiview-review-gen-swagger.yml]: Updated in the pipeline and in this PR

**Review by praveenkuttappan — APPROVED** [review]

**Pull request merged**